### ISSUE TITLE
Error, connection refused 

### ISSUE THREAD

**Issue by jayvin100** [aaronsace/MultiPoolMiner#598]
![untitled](https://user-images.githubusercontent.com/34394301/33794886-797af478-dd0f-11e7-81bc-f81afb14fdcb.png)

I'm from nicehash and have no clue what to do.

thanks!

EDIT: even happening on Nanashi 
![1](https://user-images.githubusercontent.com/34394301/33794904-d711f67c-dd0f-11e7-9cac-4ef393635f25.png)

EDIT 2: now its working on Nanashi. I think it connected to another port, 17025
![2](https://user-images.githubusercontent.com/34394301/33794931-5d7b8764-dd10-11e7-9bc6-f691aa87b7ce.png)

EDIT 3: Providing more errors.
![3](https://user-images.githubusercontent.com/34394301/33794958-e3ee5a88-dd10-11e7-866d-893fb8180206.png)

**Comment by zxhaxdr** [comment]
Zpool stratum stopped working (since hours ago) and probably the manager is sleeping. The only thing you can do is wait until Zpool back alive.

**Comment by jayvin100** [comment]
@zxhaxdr so nothing to do on my part?

**Comment by zxhaxdr** [comment]
@jayvin100 Nothing you can do. I noticed that Zpool's stratum just came back and it's working now. You can test it again. However, the status API and website are still not working properly.
If you do not want to wait for Zpool, you can switch to other similar pools such as Ahashpool and HashRefinery.
To change pool simply edit -poolname in start.bat.

**Comment by jayvin100** [comment]
@zxhaxdr oh okay, thanks!

**Comment by fonyo** [comment]
I know this is not the best answer and not exactly what you are looking for but the influx of miners seeking another place (pool) to mine on is humongous and all pools are currently struggling with the new demands/load. I recommend picking *miningpoolhub,miningpoolhubcoins* when setting up MPM. You need to create an account on MiningPoolHub but it is worth it. Please do let me know if you have other questions/queries.

**Comment by feernandobraga** [comment]
I had my miner doing that for hours yesterday before I saw and fix it. Maybe would be nice to have a feature that checks if the pool is offline (pinging its address maybe?) before starting the miner. Or even switching the algo in case the pool goes offline after the miner started.

**Comment by aaronsace** [comment]
@feernandobraga Did you not enable watchdog. 

Going to add pinging but more for finding the closest server.

**Comment by feernandobraga** [comment]
Hi @aaronsace, I have the tag -watchdog (which comes by default). Does it need a special parameter to work?

**Comment by aaronsace** [comment]
No, it should just work. 

Those screen shots have zero hash rate. Does yours have zero hash rate?

**Comment by aaronsace** [comment]
Have you got the log?

**Comment by feernandobraga** [comment]
I can't remember the hash rate information. I wasn't at home, so I just checked everthing remotely and I was a bit dipsy :P This is the log that I "think" is the right one. Hope it helps
[2017-12-09_17-48-45.txt](https://github.com/aaronsace/MultiPoolMiner/files/1545362/2017-12-09_17-48-45.txt)

**Comment by aaronsace** [comment]
That one looks fine. 

```
     Speed Active                      Launched Command
     ----- ------                      -------- -------
16.78 GH/s 00 Days 02 Hours 52 Minutes Once     Bin\NVIDIA-Palgin\ccminer.exe -a blake2s -o stratum+tcp://blake2s.mine.zpool.ca:5766 -u 15nPKyBS31cuGZsNtXvydz5t89hPQ216nk -p feerico,c=BTC
```
